### ISSUE TITLE
CSS property names are invisible in the 'Styles' tab of developer tools when using High Contrast mode

### ISSUE THREAD

**Issue by 36degrees** [alphagov/reported-bugs#55]
**Upstream bug:** N/A – reported by sending feedback through the browser
**Date:** 2020-12-23
**Reported by:** Oliver Byford
**Related to:** N/A

---

### Overview

CSS property names are invisible in the 'Styles' tab of developer tools when using Windows High Contrast with the 'High Contrast Black' or 'High Contrast White' theme

![Screenshot 2020-12-23 at 13 41 56](https://user-images.githubusercontent.com/121939/103001308-a38d0780-4524-11eb-88b9-cec6e37e11d6.png)

**Comment by querkmachine** [comment]
Not sure when it happened, but as of Edge 111, this appears to have been fixed! 🎉
